### ISSUE TITLE
stderr / stdout closed when using uvloop and ainput

### ISSUE THREAD

**Issue by mjpieters** [vxgmichel/aioconsole#117]
When using [`uvloop`](https://github.com/MagicStack/uvloop), stdout and stderr are closed when the async loop ends.

Here is a quick reproducer script:

```python
import asyncio, sys, uvloop
from aioconsole import ainput

async def test_ainput() -> str:
    return await ainput("What is your name? ")

def main(*args: str) -> None:
    run = asyncio.run
    if "--uvloop" in args:
        print("Using uvloop")
        run = uvloop.run
    name = run(test_ainput())
    print(f"Hello, {name}!")

if __name__ == "__main__":
    main(*sys.argv[1:])
```

Output, without `--uvloop` and with `--uvloop`:

```
$ ainputtest.py
What is your name? World
Hello, World!
$ ainputtest.py --uvloop
Using uvloop
What is your name? World
object address  : 0x7fb1152b2aa0
object refcount : 2
object type     : 0x7fb116acfd40
object type name: ValueError
object repr     : ValueError('I/O operation on closed file.')
lost sys.stderr
```

The `object address`, etc output is from Python's default [`sys.excepthook()`](https://docs.python.org/3/library/sys.html#sys.excepthook) implementation; if you replace the hook with one that writes the exception to a file, the traceback points to the `print()` line with the same `ValueError: I/O operation on closed file.` exception.

**Comment by vxgmichel** [comment]
Thanks @mjpieters for your valuable report :)

PR #118 should fix that. I also parametrized the tests to run with uvloop, so I can catch regressions in the future.

Feel free to leave a review if you want to have a look, otherwise I'll just go ahead and merge.

**Comment by vxgmichel** [comment]
Fixed in [v0.7.1](https://github.com/vxgmichel/aioconsole/releases/tag/v0.7.1) :tada:
